helm_deploy/values-dev.yaml

```diff
@@ -19,6 +19,7 @@ generic-service:
     DPS_URL: "https://digital-dev.prison.service.justice.gov.uk"
     FRONTEND_COMPONENT_API_URL: "https://frontend-components-dev.hmpps.service.justice.gov.uk"
     WORK_AND_INTERESTS_TAB_ENABLED: "true"
+    FRONTEND_COMPONENTS_API_FEATURE_TOGGLE_ENABLED: "true"
```

Comment:
I would leave this as false in dev for the time being, until we are ready to switch it on (the same feature toggle will be used for the header (different jira)); and from a testing point of view I think it would be better if we could say to the team "right; the new DPS header and footer are enabled - please test it"

Reply:
Fixed